### PR TITLE
Update integrate-sensitive-information-rules.md

### PR DESCRIPTION
**Pull request by kellybos**
Should link to sensitive information type definitions in SCC, not Exchange server.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/integrate-sensitive-information-rules.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ The rules can also be used in the exception part of a rule definition. Their use
 
 [Data loss prevention](data-loss-prevention.md)
 
-[Sensitive information types in Exchange Server](../../../ExchangeServer/policy-and-compliance/data-loss-prevention/sensitive-information-types.md)
+[Sensitive information types entity definitions] (https://docs.microsoft.com/en-us/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/data-loss-prevention/integrate-sensitive-information-rules.md:53]:
```suggestion
[Sensitive information type entity definitions] (https://docs.microsoft.com/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**